Components/Button.ux

```diff
@@ -0,0 +1,71 @@
+<Panel ux:Class="Material.Button" Alignment="Center" Enabled="true" Raised="true" Height="48" MinWidth="88" Padding="8,0" HitTestMode="LocalBoundsAndChildren" BackgroundColor="White" InkColor="#ccc">
+
+	<string ux:Property="Text" />
+	<bool ux:Property="Enabled" />
+	<bool ux:Property="Raised" />
+	<float4 ux:Property="BackgroundColor"/>
+	<float4 ux:Property="InkColor"/>
+
+	<Rectangle ux:Name="RaisedButton_body" ZOffset="1" Height="36" Padding="8,0" CornerRadius="2">
+		<SolidColor Color="{ReadProperty this.BackgroundColor}" />
```

Comment:
You don't need to prefix these properties with an explicit `this.*`, since they are referring to the whole `ux:Class` we are currently in and `this.*` is implied.